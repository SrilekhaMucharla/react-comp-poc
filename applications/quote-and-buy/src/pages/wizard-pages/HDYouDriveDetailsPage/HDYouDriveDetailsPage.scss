@import "../../../theme/theme.scss";

.you-drive-cover-details {
    .steps-message {
        .label-text {
            text-align: center;
        }
    }
}

.you-drive-cover-details__control-app-image {
    height: 130px;
}

.you-drive-cover-details__control-key-image {
    padding: 1.5em;
}

.you-drive-cover-details__control-car-image {
    padding: 0.6em;
}

.you-drive-cover-details__container {
    display: flex;
    border-bottom: 1px solid $blue-light;

    @include media-breakpoint-down(sm) {
        flex-direction: column-reverse;
    }
}

.you-drive-cover-details__benefits-list {
    .size-md {
        color: $blue-dark-3;
        line-height: 24.3px;
        font-family: $font-face-generic-regular;
        font-weight: 600;
    }
}

.you-drive-cover-details__benefits-list-point {
    display: inherit;
}

.you-drive-cover-details__benefits-list-last-point {
    padding-bottom: 15px;
}

.you-drive-cover-details__benefits-messages {
    padding: 30px 30px 48px;
    @include media-breakpoint-down(sm) {
        margin-bottom: -70px;
        padding: 50px 20px 20px;
    }
    @include media-breakpoint-down(md) {
        padding: 20px;
    }
}

.you-drive-cover-details__benefits-header {
    display: flex;
    align-items: baseline;
}

.you-drive-cover-details__benefits-dash {
    padding-left: 2em;
}

.you-drive-cover-details__benefits-message {
    padding-left: 1em;
}

.you-drive-cover-details__control-mask {
    text-align: end;
    &::before {
        content: "";
        background-color: $midnight-royal-blue;
        position: absolute;
        width: 100%;
        height: 299px;
        @include media-breakpoint-up(md) {
            width: 200px;
        }
    }
}
.you-drive-cover-details__image-desktop {
    z-index: 1;
    width: 100%;
    clip-path: ellipse(438px 214px at 66.5% 0px);
    @include media-breakpoint-up(sm) {
        clip-path: ellipse(495px 286px at 66.5% 0px);
    }
    @include media-breakpoint-up(md) {
        width: auto;
        clip-path: ellipse(350px 374px at 96% 0);
        height: 100%;
    }
}

.you-drive-cover-detail__benefits-subheader {
    margin-bottom: 20px;
}

.you-drive-cover-details__image {
    clip-path: ellipse(506px 219px at 65% 0);
    @include media-breakpoint-up(sm) {
        clip-path: ellipse(594px 290px at 65% 0);
    }
    @include media-breakpoint-up(md) {
        clip-path: ellipse(401px 374px at 100.5% 0px);
    }
}
