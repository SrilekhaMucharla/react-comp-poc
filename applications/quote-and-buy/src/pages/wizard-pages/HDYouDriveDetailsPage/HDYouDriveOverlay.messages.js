/* eslint-disable max-len */
export const headerMessage = 'Why YouDrive?';
export const overlayInfoOneMesage = 'Get a lower price from day one by easily sharing your driving data through our app and tab technology';

export const benefitsOneHeader = 'Simple';
export const benefitsTwoHeader = 'Great value';
export const benefitsThreeHeader = "You're in control";
export const benefitsOneMessage = "Your driving data is shared automatically through the app (you don't have to do anything). We can use the data to help you against fraudulent claims. This runs in the background so there's no need to open it each time you take a trip.";
export const benefitsTwoMessage = "Most drivers are good drivers, so we give you a great price for choosing to share your driving data. Just keep driving well and you'll get great prices year after year.";
export const benefitsThreeMessage = 'You can start driving from the moment your policy starts. Learn how your driving impacts your score as you go. When your wireless tab arrives, just follow the steps below.';


export const overlayInfoTwoHeader = 'How it all works';
export const appHeader = 'Download the app';
export const keyHeader = 'Set up wireless tab';
export const carHeader = 'Drive';
export const appMessage = 'Download the Hastings Direct app to your smartphone. It only uses a small amount of battery and you can save data by using wi-fi only sharing.';
export const keyMessage = "Your tab will be posted to your home address. When it arrives (it's about the same size as your keyfob), pop it in your car and pair it with your smartphone.";
export const carMessage = "You'll see exactly how you're doing in the Hastings Direct app, with the app and tab picking up things like speeding and using your phone while driving.";
export const benefitOneMessageOne = '60 second set-up';
export const benefitOneMessageTwo = 'Works automatically';
export const benefitOneMessageThree = 'No time of day penalties';
export const benefitTwoMessageOne = 'Great price today';
export const benefitTwoMessageTwo = 'Ongoing low prices for good drivers';
export const benefitThreeMessageOne = 'See the data we see';
export const benefitThreeMessageTwo = 'Manage your policy in the app';
export const benefitThreeMessageThree = 'Helps protect against fraudulent claims';
export const customizeQuote = 'Customise quote - YouDrive';
