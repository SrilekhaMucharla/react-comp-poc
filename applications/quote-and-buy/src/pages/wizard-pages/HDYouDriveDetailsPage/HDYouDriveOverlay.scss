#you-drive-overlay {
  .overlay-container {
    padding: 0;
  }
}
.you-drive-overlay {
  margin: 5rem 3rem;
  @include media-breakpoint-down(sm) {
    margin: 3rem 2rem;
  }

  .label-text {
    line-height: 1.5;
  }
}

.you-drive-overlay__you-drive-icon {
  width: 185px;
}

.you-drive-overlay__overlay-header {
  margin-left: 0;
  padding-top: 50px;
  padding-left: 0;
  font-size: 25px;
  line-height: 30px;
  @include media-breakpoint-down(sm) {
    padding-top: 25px;
  }
  .label-text {
    font-weight: bold;
    font-size: $font-size-xl !important;
  }
}

.you-drive-overlay__overlay-message {
  margin-bottom: 30px;
  margin-top: -15px;

  @media (max-width: 767px) {
    margin-bottom: 0;
  }

  .label-text {
    color: $mono-blue-yonder;
  }
}

.you-drive-overlay__overlay-list {
  .you-drive-overlay__overlay-message {
    margin-left: 10px;
  }

  .dash-icon {
    margin-left: 15px;
    margin-right: 10px;
  }

  .label-icon-left {
    .img {
      width: 30px;
      height: 30px;
    }
  }
}

.you-drive-overlay__overlay-sub-header {
  margin: 55px 0 50px 0;
  .label-text {
    font-weight: bold;
  }
}

.you-drive-overlay__paragraph-header {
  .label-text {
    font-weight: bold;
    display: flex;
    .label-icon-left {
      height: unset;
      margin-top: -3px;
      padding-right: 20px;
    }
  }
}

.you-drive-overlay__paragraph-tick {
  height: 24px;
  width: 24px;
}

.you-drive-overlay__you-drive-step {
  display: flex;
  margin: 1em 0em;
}

.you-drive-overlay__steps-image {
  align-items: flex-start;
  display: flex;
  min-width: 125px;
  align-self: center;

  img {
    width: 100%;
    margin: auto;
    max-height: 150px;
    max-width: 150px;
  }
  .you-drive-overlay__car-image,
  .you-drive-overlay__key-image {
    width: 110%;
    position: relative;
    left: -10px;
  }
}

.you-drive-overlay__steps-description {
  display: flex;
  flex-direction: column;
  margin-left: 2em;
  @include media-breakpoint-down(sm) {
    margin: 0 10px;
  }
}

.you-drive-overlay__steps-header {
  @include media-breakpoint-down(md) {
    justify-content: center;
  }

  display: flex;
  align-items: center;
  .label-text {
    font-weight: bold;
  }
  span {
    margin-bottom: 0;
  }
}

.you-drive-overlay__steps-message {
  @include media-breakpoint-up(md) {
    .label-text {
      text-align: left;
    }
  }
}

.you-drive-overlay__label {
  align-items: center;
}
.you-drive-overlay__header-label {
  font-size: $hd-font-size-h2-mobile;
}