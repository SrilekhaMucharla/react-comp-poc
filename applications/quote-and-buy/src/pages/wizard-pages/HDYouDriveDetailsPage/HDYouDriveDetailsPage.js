import React from 'react';
import { Row, Col } from 'react-bootstrap';
import {
    HDLabelRefactor
} from 'hastings-components';
import { withRouter } from 'react-router-dom';
import PropTypes from 'prop-types';
import Tick from '../../../assets/images/icons/tick-icon.svg';
import watchingMobile from '../../../assets/images/wizard-images/static-youdrive-phone-sea-001.jpg';
import mobileNavigation from '../../../assets/images/icons/static-youdrive-journey-phone-001.png';
import keyImage from '../../../assets/images/wizard-images/static-youdrive-tab-keyfob-001.png';
import redCarImage from '../../../assets/images/wizard-images/static-red-car-001.png';
import circleNumberOne from '../../../assets/images/wizard-images/circle-number-1.svg';
import circleNumberTwo from '../../../assets/images/wizard-images/circle-number-2.svg';
import circleNumberThree from '../../../assets/images/wizard-images/circle-number-3.svg';
import * as messages from './HDYouDriveDetailsPage.messages';
// import './HDYouDriveDetailsPage.scss';
import HDYouDriveOverlay from './HDYouDriveOverlay';
import { pageMetadataPropTypes } from '../../../constant/propTypes';


export const HDYouDriveDetailsPage = ({ pageMetadata }) => {
    return (
        <Row className="you-drive-cover-details">
            <Col>
                <Row className="you-drive-cover-details__container text-left">
                    <Col className="you-drive-cover-details__benefits-messages">
                        <Row className="you-drive-cover-details__benefits-header mb-3">
                            <Col>
                                <HDLabelRefactor
                                    className="you-drive-cover-details__benefits-header-label"
                                    Tag="h5"
                                    size="md"
                                    text={messages.headerMessage} />
                            </Col>
                            <Col className="text-left px-0 col-2">
                                <HDYouDriveOverlay pageMetadata={pageMetadata} />
                            </Col>
                        </Row>
                        <Row>
                            <Col>
                                <HDLabelRefactor
                                    className="you-drive-cover-details__benefits-header-info"
                                    Tag="p"
                                    text={messages.messageText} />
                            </Col>
                        </Row>
                        <Row>
                            <Col>
                                <HDLabelRefactor
                                    Tag="h5"
                                    text={messages.benefitOne}
                                    icon={<img src={Tick} alt="tick" />}
                                    iconPosition="l" />
                                <Row>
                                    <Col className="you-drive-cover-details__benefits-list-point">
                                        <HDLabelRefactor className="you-drive-cover-details__benefits-dash mb-1" Tag="p" text="-" />
                                        <HDLabelRefactor
                                            className="you-drive-cover-details__benefits-message mb-1"
                                            Tag="p"
                                            text={messages.benefitOneMessageOne} />
                                    </Col>
                                </Row>
                                <Row>
                                    <Col className="you-drive-cover-details__benefits-list-point">
                                        <HDLabelRefactor className="you-drive-cover-details__benefits-dash mb-1" Tag="p" text="-" />
                                        <HDLabelRefactor
                                            className="you-drive-cover-details__benefits-message mb-1"
                                            Tag="p"
                                            text={messages.benefitOneMessageTwo} />
                                    </Col>
                                </Row>
                                <Row>
                                    <Col className="you-drive-cover-details__benefits-list-point">
                                        <HDLabelRefactor className="you-drive-cover-details__benefits-dash" Tag="p" text="-" />
                                        <HDLabelRefactor className="you-drive-cover-details__benefits-message" Tag="p" text={messages.benefitOneMessageThree} />
                                    </Col>
                                </Row>
                                <HDLabelRefactor
                                    Tag="h5"
                                    size="md"
                                    text={messages.benefitTwo}
                                    icon={<img src={Tick} alt="tick" />}
                                    iconPosition="l" />
                                <Row>
                                    <Col className="you-drive-cover-details__benefits-list-point">
                                        <HDLabelRefactor className="you-drive-cover-details__benefits-dash mb-1" Tag="p" text="-" />
                                        <HDLabelRefactor
                                            className="you-drive-cover-details__benefits-message mb-1"
                                            Tag="p"
                                            text={messages.benefitTwoMessageOne} />
                                    </Col>
                                </Row>
                                <Row>
                                    <Col className="you-drive-cover-details__benefits-list-point">
                                        <HDLabelRefactor className="you-drive-cover-details__benefits-dash" Tag="p" text="-" />
                                        <HDLabelRefactor className="you-drive-cover-details__benefits-message" Tag="p" text={messages.benefitTwoMessageTwo} />
                                    </Col>
                                </Row>
                                <HDLabelRefactor
                                    Tag="h5"
                                    size="md"
                                    text={messages.benefitThree}
                                    icon={<img src={Tick} alt="tick" />}
                                    iconPosition="l" />
                                <Row>
                                    <Col className="you-drive-cover-details__benefits-list-point">
                                        <HDLabelRefactor className="you-drive-cover-details__benefits-dash mb-1" Tag="p" text="-" />
                                        <HDLabelRefactor
                                            className="you-drive-cover-details__benefits-message mb-1"
                                            Tag="p"
                                            text={messages.benefitThreeMessageOne} />
                                    </Col>
                                </Row>
                                <Row>
                                    <Col className="you-drive-cover-details__benefits-list-point">
                                        <HDLabelRefactor className="you-drive-cover-details__benefits-dash mb-1" Tag="p" text="-" />
                                        <HDLabelRefactor
                                            className="you-drive-cover-details__benefits-message mb-1"
                                            Tag="p"
                                            text={messages.benefitThreeMessageTwo} />
                                    </Col>
                                </Row>
                                <Row>
                                    <Col className="you-drive-cover-details__benefits-list-point">
                                        <HDLabelRefactor className="you-drive-cover-details__benefits-dash mb-1" Tag="p" text="-" />
                                        <HDLabelRefactor
                                            className="you-drive-cover-details__benefits-message mb-1"
                                            Tag="p"
                                            text={messages.benefitThreeMessageThree} />
                                    </Col>
                                </Row>
                            </Col>
                        </Row>
                    </Col>
                    <Col className="you-drive-cover-details__image px-0">
                        <div className="you-drive-cover-details__control-mask">
                            <img className="you-drive-cover-details__image-desktop" src={watchingMobile} alt="" />
                        </div>
                    </Col>
                </Row>
                <Row className="you-drive-cover-details__control-benefits">
                    <Col>
                        <Row className="mt-5">
                            <Col className="text-center">
                                <HDLabelRefactor
                                    className="you-drive-cover-details__control-header"
                                    text={messages.youAteInControl}
                                    Tag="h5" />
                            </Col>
                        </Row>
                        <Row className="mb-5">
                            <Col className="text-center">
                                <HDLabelRefactor
                                    className="you-drive-cover-detail__benefits-subheader"
                                    text={messages.driveStraightAway}
                                    Tag="p" />
                            </Col>
                        </Row>
                        <Row className="d-block d-md-flex">
                            <Col className="you-drive-cover-details__control-app">
                                <Row className="you-drive-cover-details__control-steps-image">
                                    <Col className="text-center">
                                        <img className="you-drive-cover-details__control-app-image" src={mobileNavigation} alt="" />
                                    </Col>
                                </Row>
                                <Row className="you-drive-cover-details__control-app-header justify-content-center justify-content-md-start">
                                    <Col className="col-1 col-md-2">
                                        <img className="you-drive-cover-details__control-number-image" src={circleNumberOne} alt="" />
                                    </Col>
                                    <Col className="col-5 col-md-10">
                                        <HDLabelRefactor
                                            className="text-left"
                                            Tag="h5"
                                            text={messages.appHeader} />
                                    </Col>
                                </Row>
                                <Row className="you-drive-cover-details__control-app-message steps-message mt-2">
                                    <Col>
                                        <HDLabelRefactor
                                            Tag="p"
                                            text={messages.appMessage} />
                                    </Col>
                                </Row>
                            </Col>
                            <Col className="you-drive-cover-details__control-you-drive-key">
                                <Row className="you-drive-cover-details__control-steps-image">
                                    <Col className="text-center">
                                        <img className="you-drive-cover-details__control-key-image" src={keyImage} alt="" />
                                    </Col>
                                </Row>
                                <Row className="you-drive-cover-details__control-key-header justify-content-center justify-content-md-start">
                                    <Col className="col-1 col-md-2">
                                        <img className="you-drive-cover-details__control-number-image" src={circleNumberTwo} alt="" />
                                    </Col>
                                    <Col className="col-5 col-md-10">
                                        <HDLabelRefactor
                                            className="text-left"
                                            Tag="h5"
                                            text={messages.keyHeader} />
                                    </Col>
                                </Row>
                                <Row className="you-drive-cover-details__control-key-message steps-message mt-2">
                                    <Col>
                                        <HDLabelRefactor
                                            Tag="p"
                                            text={messages.keyMessage} />
                                    </Col>
                                </Row>
                            </Col>
                            <Col className="you-drive-cover-details__control-you-drive-car">
                                <Row className="you-drive-cover-details__control-steps-image">
                                    <Col className="text-center">
                                        <img className="you-drive-cover-details__control-car-image" src={redCarImage} alt="" />
                                    </Col>
                                </Row>
                                <Row className="you-drive-cover-details__control-car-header justify-content-center justify-content-md-start">
                                    <Col className="col-1 col-md-2">
                                        <img className="you-drive-cover-details__control-number-image" src={circleNumberThree} alt="" />
                                    </Col>
                                    <Col className="col-4 col-md-10">
                                        <HDLabelRefactor
                                            className="text-left"
                                            Tag="h5"
                                            text={messages.carHeader} />
                                    </Col>
                                </Row>
                                <Row className="you-drive-cover-details__control-car-message steps-message mt-2">
                                    <Col>
                                        <HDLabelRefactor
                                            Tag="p"
                                            text={messages.carMessage} />
                                    </Col>
                                </Row>
                            </Col>
                        </Row>
                    </Col>
                </Row>
            </Col>
        </Row>
    );
};

HDYouDriveDetailsPage.propTypes = {
    pageMetadata: PropTypes.shape(pageMetadataPropTypes).isRequired
};

export default withRouter(HDYouDriveDetailsPage);
