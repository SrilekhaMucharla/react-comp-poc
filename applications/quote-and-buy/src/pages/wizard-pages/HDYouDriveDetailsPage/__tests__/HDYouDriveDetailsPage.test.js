import React from 'react';
import { mount, shallow } from 'enzyme';
import { HDYouDriveDetailsPage } from '../HDYouDriveDetailsPage';

describe('<HDYouDriveDetailsPage />', () => {
    test('render component', () => {
        const wrapper = shallow(
            <HDYouDriveDetailsPage />
        );
        expect(wrapper).toMatchSnapshot();
    });
    test('benefits-list not disabled', () => {
        const wrapper = mount(
            <HDYouDriveDetailsPage />
        );
        expect(wrapper.find('.you-drive-cover-details').at(0).prop('disabled')).toBeFalsy();
    });
    test('benefits-list contains children', () => {
        const wrapper = mount(
            <HDYouDriveDetailsPage />
        );
        expect(wrapper.find('.you-drive-cover-details__benefits-message').at(0).children().at(0)
            .hasClass('label-text'));
    });
    test('Overlay is not present', () => {
        const wrapper = mount(
            <HDYouDriveDetailsPage />
        );
        expect(wrapper.find('.you-drive-overlay').exists()).toBeFalsy();
    });
});
