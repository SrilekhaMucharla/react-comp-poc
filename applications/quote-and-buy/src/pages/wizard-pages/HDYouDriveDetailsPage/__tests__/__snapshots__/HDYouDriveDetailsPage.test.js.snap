// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HDYouDriveDetailsPage /> render component 1`] = `
<Row
  className="you-drive-cover-details"
  noGutters={false}
>
  <Col>
    <Row
      className="you-drive-cover-details__container text-left"
      noGutters={false}
    >
      <Col
        className="you-drive-cover-details__benefits-messages"
      >
        <Row
          className="you-drive-cover-details__benefits-header mb-3"
          noGutters={false}
        >
          <Col>
            <HDLabelRefactor
              Tag="h5"
              adjustImagePosition={true}
              className="you-drive-cover-details__benefits-header-label"
              size="md"
              text="Why YouDrive is different"
            />
          </Col>
          <Col
            className="text-left px-0 col-2"
          >
            <HDYouDriveOverlay />
          </Col>
        </Row>
        <Row
          noGutters={false}
        >
          <Col>
            <HDLabelRefactor
              Tag="p"
              adjustImagePosition={true}
              className="you-drive-cover-details__benefits-header-info"
              text="Get a lower price from day one by easily sharing your driving data through our app and tab technology"
            />
          </Col>
        </Row>
        <Row
          noGutters={false}
        >
          <Col>
            <HDLabelRefactor
              Tag="h5"
              adjustImagePosition={true}
              className=""
              icon={
                <img
                  alt="tick"
                  src="tick-icon.svg"
                />
              }
              iconPosition="l"
              text="Simple"
            />
            <Row
              noGutters={false}
            >
              <Col
                className="you-drive-cover-details__benefits-list-point"
              >
                <HDLabelRefactor
                  Tag="p"
                  adjustImagePosition={true}
                  className="you-drive-cover-details__benefits-dash mb-1"
                  text="-"
                />
                <HDLabelRefactor
                  Tag="p"
                  adjustImagePosition={true}
                  className="you-drive-cover-details__benefits-message mb-1"
                  text="60 second set-up"
                />
              </Col>
            </Row>
            <Row
              noGutters={false}
            >
              <Col
                className="you-drive-cover-details__benefits-list-point"
              >
                <HDLabelRefactor
                  Tag="p"
                  adjustImagePosition={true}
                  className="you-drive-cover-details__benefits-dash mb-1"
                  text="-"
                />
                <HDLabelRefactor
                  Tag="p"
                  adjustImagePosition={true}
                  className="you-drive-cover-details__benefits-message mb-1"
                  text="Works automatically"
                />
              </Col>
            </Row>
            <Row
              noGutters={false}
            >
              <Col
                className="you-drive-cover-details__benefits-list-point"
              >
                <HDLabelRefactor
                  Tag="p"
                  adjustImagePosition={true}
                  className="you-drive-cover-details__benefits-dash"
                  text="-"
                />
                <HDLabelRefactor
                  Tag="p"
                  adjustImagePosition={true}
                  className="you-drive-cover-details__benefits-message"
                  text="No time of day penalties"
                />
              </Col>
            </Row>
            <HDLabelRefactor
              Tag="h5"
              adjustImagePosition={true}
              className=""
              icon={
                <img
                  alt="tick"
                  src="tick-icon.svg"
                />
              }
              iconPosition="l"
              size="md"
              text="Great value"
            />
            <Row
              noGutters={false}
            >
              <Col
                className="you-drive-cover-details__benefits-list-point"
              >
                <HDLabelRefactor
                  Tag="p"
                  adjustImagePosition={true}
                  className="you-drive-cover-details__benefits-dash mb-1"
                  text="-"
                />
                <HDLabelRefactor
                  Tag="p"
                  adjustImagePosition={true}
                  className="you-drive-cover-details__benefits-message mb-1"
                  text="Great price today"
                />
              </Col>
            </Row>
            <Row
              noGutters={false}
            >
              <Col
                className="you-drive-cover-details__benefits-list-point"
              >
                <HDLabelRefactor
                  Tag="p"
                  adjustImagePosition={true}
                  className="you-drive-cover-details__benefits-dash"
                  text="-"
                />
                <HDLabelRefactor
                  Tag="p"
                  adjustImagePosition={true}
                  className="you-drive-cover-details__benefits-message"
                  text="Ongoing low prices for good drivers"
                />
              </Col>
            </Row>
            <HDLabelRefactor
              Tag="h5"
              adjustImagePosition={true}
              className=""
              icon={
                <img
                  alt="tick"
                  src="tick-icon.svg"
                />
              }
              iconPosition="l"
              size="md"
              text="You're in control"
            />
            <Row
              noGutters={false}
            >
              <Col
                className="you-drive-cover-details__benefits-list-point"
              >
                <HDLabelRefactor
                  Tag="p"
                  adjustImagePosition={true}
                  className="you-drive-cover-details__benefits-dash mb-1"
                  text="-"
                />
                <HDLabelRefactor
                  Tag="p"
                  adjustImagePosition={true}
                  className="you-drive-cover-details__benefits-message mb-1"
                  text="See the data we see"
                />
              </Col>
            </Row>
            <Row
              noGutters={false}
            >
              <Col
                className="you-drive-cover-details__benefits-list-point"
              >
                <HDLabelRefactor
                  Tag="p"
                  adjustImagePosition={true}
                  className="you-drive-cover-details__benefits-dash mb-1"
                  text="-"
                />
                <HDLabelRefactor
                  Tag="p"
                  adjustImagePosition={true}
                  className="you-drive-cover-details__benefits-message mb-1"
                  text="Manage your policy in the app"
                />
              </Col>
            </Row>
            <Row
              noGutters={false}
            >
              <Col
                className="you-drive-cover-details__benefits-list-point"
              >
                <HDLabelRefactor
                  Tag="p"
                  adjustImagePosition={true}
                  className="you-drive-cover-details__benefits-dash mb-1"
                  text="-"
                />
                <HDLabelRefactor
                  Tag="p"
                  adjustImagePosition={true}
                  className="you-drive-cover-details__benefits-message mb-1"
                  text="Helps protect against fraudulent claims"
                />
              </Col>
            </Row>
          </Col>
        </Row>
      </Col>
      <Col
        className="you-drive-cover-details__image px-0"
      >
        <div
          className="you-drive-cover-details__control-mask"
        >
          <img
            alt=""
            className="you-drive-cover-details__image-desktop"
            src="static-youdrive-phone-sea-001.jpg"
          />
        </div>
      </Col>
    </Row>
    <Row
      className="you-drive-cover-details__control-benefits"
      noGutters={false}
    >
      <Col>
        <Row
          className="mt-5"
          noGutters={false}
        >
          <Col
            className="text-center"
          >
            <HDLabelRefactor
              Tag="h5"
              adjustImagePosition={true}
              className="you-drive-cover-details__control-header"
              text="Youâ€™re in control"
            />
          </Col>
        </Row>
        <Row
          className="mb-5"
          noGutters={false}
        >
          <Col
            className="text-center"
          >
            <HDLabelRefactor
              Tag="p"
              adjustImagePosition={true}
              className="you-drive-cover-detail__benefits-subheader"
              text="You can start driving straight away. Then, when your tab arrives, here's what to do..."
            />
          </Col>
        </Row>
        <Row
          className="d-block d-md-flex"
          noGutters={false}
        >
          <Col
            className="you-drive-cover-details__control-app"
          >
            <Row
              className="you-drive-cover-details__control-steps-image"
              noGutters={false}
            >
              <Col
                className="text-center"
              >
                <img
                  alt=""
                  className="you-drive-cover-details__control-app-image"
                  src="static-youdrive-journey-phone-001.png"
                />
              </Col>
            </Row>
            <Row
              className="you-drive-cover-details__control-app-header justify-content-center justify-content-md-start"
              noGutters={false}
            >
              <Col
                className="col-1 col-md-2"
              >
                <img
                  alt=""
                  className="you-drive-cover-details__control-number-image"
                  src="circle-number-1.svg"
                />
              </Col>
              <Col
                className="col-5 col-md-10"
              >
                <HDLabelRefactor
                  Tag="h5"
                  adjustImagePosition={true}
                  className="text-left"
                  text="Download the app"
                />
              </Col>
            </Row>
            <Row
              className="you-drive-cover-details__control-app-message steps-message mt-2"
              noGutters={false}
            >
              <Col>
                <HDLabelRefactor
                  Tag="p"
                  adjustImagePosition={true}
                  className=""
                  text="Download the Hastings Direct app to your smartphone. It only uses a small amount of battery and you can save data by using wi-fi only sharing."
                />
              </Col>
            </Row>
          </Col>
          <Col
            className="you-drive-cover-details__control-you-drive-key"
          >
            <Row
              className="you-drive-cover-details__control-steps-image"
              noGutters={false}
            >
              <Col
                className="text-center"
              >
                <img
                  alt=""
                  className="you-drive-cover-details__control-key-image"
                  src="static-youdrive-tab-keyfob-001.png"
                />
              </Col>
            </Row>
            <Row
              className="you-drive-cover-details__control-key-header justify-content-center justify-content-md-start"
              noGutters={false}
            >
              <Col
                className="col-1 col-md-2"
              >
                <img
                  alt=""
                  className="you-drive-cover-details__control-number-image"
                  src="circle-number-2.svg"
                />
              </Col>
              <Col
                className="col-5 col-md-10"
              >
                <HDLabelRefactor
                  Tag="h5"
                  adjustImagePosition={true}
                  className="text-left"
                  text="Set up wireless tab"
                />
              </Col>
            </Row>
            <Row
              className="you-drive-cover-details__control-key-message steps-message mt-2"
              noGutters={false}
            >
              <Col>
                <HDLabelRefactor
                  Tag="p"
                  adjustImagePosition={true}
                  className=""
                  text="Your tab will be posted to your home address. When it arrives (it's about the same size as your keyfob), pop it in your car and pair it with your smartphone."
                />
              </Col>
            </Row>
          </Col>
          <Col
            className="you-drive-cover-details__control-you-drive-car"
          >
            <Row
              className="you-drive-cover-details__control-steps-image"
              noGutters={false}
            >
              <Col
                className="text-center"
              >
                <img
                  alt=""
                  className="you-drive-cover-details__control-car-image"
                  src="static-red-car-001.png"
                />
              </Col>
            </Row>
            <Row
              className="you-drive-cover-details__control-car-header justify-content-center justify-content-md-start"
              noGutters={false}
            >
              <Col
                className="col-1 col-md-2"
              >
                <img
                  alt=""
                  className="you-drive-cover-details__control-number-image"
                  src="circle-number-3.svg"
                />
              </Col>
              <Col
                className="col-4 col-md-10"
              >
                <HDLabelRefactor
                  Tag="h5"
                  adjustImagePosition={true}
                  className="text-left"
                  text="Drive"
                />
              </Col>
            </Row>
            <Row
              className="you-drive-cover-details__control-car-message steps-message mt-2"
              noGutters={false}
            >
              <Col>
                <HDLabelRefactor
                  Tag="p"
                  adjustImagePosition={true}
                  className=""
                  text="You'll see exactly how you're doing in the Hastings Direct app, with the tab picking up things like speeding and using your phone while driving."
                />
              </Col>
            </Row>
          </Col>
        </Row>
      </Col>
    </Row>
  </Col>
</Row>
`;
